plio/permissions.py

```diff
@@ -0,0 +1,34 @@
+from rest_framework import permissions
+from organizations.middleware import OrganizationTenantMiddleware
+from plio.settings import DEFAULT_TENANT_SHORTCODE
+from users.models import OrganizationUser
+
+
+class PlioPermission(permissions.BasePermission):
+    """
+    Permission check for plios.
+    """
+
+    def has_permission(self, request, view):
+        return True
+
+    def has_object_permission(self, request, view, obj):
+        if request.user.is_superuser:
+            return True
+
+        organization_shortcode = (
+            OrganizationTenantMiddleware.get_organization_shortcode(request)
+        )
+        if organization_shortcode == DEFAULT_TENANT_SHORTCODE:
```

Comment:
one line comment would be helpful - saying that when the plio is in the personal workspace, only the one who created it should be able to access.